site/docs/.vuepress/styles/index.scss

```diff
@@ -6,9 +6,52 @@
 }
 
 html.dark {
+  --c-bg-lighter: #2b313a;
   --c-brand-light: #138cb3;
-  --c-tip: #1583a7;
   --c-brand: #009dca;
+  --c-tip: #1583a7;
+  --c-tip-title: var(--c-text-accent);
+  --c-warning: #e0ad15;
+  --c-warning-bg: rgba(255, 197, 23, .05);
+  --c-warning-title: #fdca31;
+  --c-warning-text: rgba(252, 252, 124, 0.9);
+  --c-danger: #fc1e38;
+  --c-danger-bg: rgba(237, 60, 80, .06);
+  --c-danger-title: #fc2d3b;
+  --c-danger-text: rgb(253, 128, 136);
+}
+
+html.dark .custom-container.warning code{
+  color: rgba(252, 210, 83, 0.9);
+  background-color: rgba(255, 197, 23, .1);
+}
+
+html.dark .custom-container.danger code{
+  color: rgb(252, 83, 83);
+  background-color: rgba(237, 60, 80, .15);
+}
```

Comment:
These changes to code blocks definitely add a nice touch, but you forgot the light theme too. I suggest moving it to a separate PR